R/lime.R

```diff
@@ -159,7 +159,7 @@ select_tree <- function(x, y, weights, n_features) {
 #' @importFrom stats coef
 select_f_lp <- function(x, y, weights, n_features) {
   shuffle_order <- sample(length(y)) # glm is sensitive to the order of the examples
-  fit <- glmnet(x[shuffle_order,], y[shuffle_order], weights = weights[shuffle_order], alpha = 1, nlambda = 300)
+  fit <- glmnet(x[shuffle_order,], y[shuffle_order], weights = weights[shuffle_order], alpha = 1, nlambda = 3000)
   # In case that no model with correct n_feature size was found
   if (all(fit$df != n_features)) {
```

Comment:
Okay. I think I understand what is causing the problem now... @thomasp85 do you think it would be better to return less coefficients than the user is asking for when there aren't exactly what they need or to just allow it to drop the extras randomly as it would if we just comment out the lines 164-166?